src/bsoncxx/include/bsoncxx/v_noabi/bsoncxx/types/bson_value/view.hpp

```diff
@@ -306,32 +307,33 @@ class BSONCXX_API view {
     };
 };
 
-// sfinae in the bool return to avoid competing with the value == value
-// operators.
 template <typename T>
-using not_view = typename std::enable_if<
-    std::is_constructible<bson_value::view, T>::value &&
-        !std::is_same<typename std::decay<T>::type, bson_value::view>::value &&
-        !std::is_same<typename std::decay<T>::type, bson_value::value>::value,
-    bool>::type;
+using is_bson_view_compatible =
```

Comment:
```suggestion
using not_view =
```

Suggest renaming back to `not_view` to preserve the existing public API.

Reply:
Restored a `not_view` alias.